.ci/esy-build-steps.yml

```diff
@@ -10,6 +10,13 @@ steps:
 
   - script: 'esy test'
     displayName: 'esy test'
+  
+  - task: PublishTestResults@2
+    displayName: 'Publish JUnit file'
+    inputs:
+      testResultsFormat: JUnit
+      testResultsFiles: './junit.xml'
+      failTaskOnFailedTests: true
```

Comment:
This way we can let the tests fail but still upload the result

Reply:
Nice, is there nice GH integration possible with like commenting on PR's with the output from failing tests? I looked at https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-test-results?view=azure-devops&tabs=yaml and didn't see anything like that, but it seems like a natural thing to exist

Reply:
When the run is done you can see the results here: https://github.com/facebookexperimental/reason-native/pull/128/checks?check_run_id=80521129

Reply:
Nice, this is really cool! https://dev.azure.com/reason-native/reason-native/_build/results?buildId=414&view=ms.vss-test-web.build-test-results-tab